tls/s2n_cipher_preferences.c

```diff
@@ -1782,6 +1858,7 @@ struct s2n_cipher_suite *cipher_suites_rfc9151[] = {
 const struct s2n_cipher_preferences cipher_preferences_rfc9151 = {
     .count = s2n_array_len(cipher_suites_rfc9151),
     .suites = cipher_suites_rfc9151,
+    .allow_chacha20_boosting = false,
 };
```

Comment:
allow_chacha20_boosting will actually be set to "false" if not included in the definition. Are we including the new field for visibility?

Reply:
Yes. I thought it better to be explicit than rely on default initializations